.circleci/config.yml

```diff
@@ -3,86 +3,79 @@
 ### https://circleci.com/docs/config-sample
 ### =============================================================
 version: 2.1
+orbs:
+  python: circleci/python@<< pipeline.parameters.python-orb-version >>
+  node: circleci/node@<< pipeline.parameters.node-orb-version >>
+  orb-tools: circleci/orb-tools@10.0
+
+parameters:
+  python-orb-version:
+    type: string
+    default: "2.0.3"
+  node-orb-version:
+    type: string
+    default: "5.0.1"
+  python-cache-version:
+    type: string
+    default: v3
+  dev-cache-version:
+    type: string
+    default: v3
+  node-cache-version:
+    type: string
+    default: v3
+  global-docker-image:
+    type: string
+    default: cimg/python:3.9.10-node
```

Comment:
3.10?

Reply:
https://github.com/demisto/demisto-sdk/pull/1963/commits/ff8bf2595b484ac7d03294ebb4cefc0921ef82ad#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R23

Reply:
@dorschw, he was next to me as I was reviewing so he committed some CR changes before I technically submitted my review.